x-pack/plugins/observability_ai_assistant/public/components/chat/chat_header.tsx

```diff
@@ -30,36 +33,54 @@ const chatHeaderClassName = css`
 `;
 
 export function ChatHeader({
-  title,
-  loading,
-  licenseInvalid,
   connectors,
   conversationId,
+  flyoutWidthMode,
+  licenseInvalid,
+  loading,
   showLinkToConversationsApp,
+  title,
   onCopyConversation,
   onSaveTitle,
+  onToggleFlyoutWidthMode,
 }: {
-  title: string;
-  loading: boolean;
-  licenseInvalid: boolean;
   connectors: UseGenAIConnectorsResult;
   conversationId?: string;
+  flyoutWidthMode?: FlyoutWidthMode;
+  licenseInvalid: boolean;
+  loading: boolean;
   showLinkToConversationsApp: boolean;
+  title: string;
   onCopyConversation: () => void;
   onSaveTitle: (title: string) => void;
+  onToggleFlyoutWidthMode?: (newFlyoutWidthMode: FlyoutWidthMode) => void;
 }) {
   const theme = useEuiTheme();
 
+  const router = useObservabilityAIAssistantRouter();
+
   const [newTitle, setNewTitle] = useState(title);
 
   useEffect(() => {
     setNewTitle(title);
   }, [title]);
 
-  const chatActionsMenuWrapper = css`
-    position: absolute;
-    right: 46px;
-  `;
+  const handleToggleFlyoutWidthMode = () => {
+    onToggleFlyoutWidthMode?.(flyoutWidthMode === 'side' ? 'full' : 'side');
+  };
+
+  const handleNavigateToConversations = () => {
+    if (conversationId) {
+      router.navigateToConversationsApp('/conversations/{conversationId}', {
+        path: {
+          conversationId,
+        },
+        query: {},
+      });
+    } else {
+      router.navigateToConversationsApp('/conversations/new', { path: {}, query: {} });
+    }
```

Comment:
Let's keep the `router` api minimal and use the existing `push`/`replace` methods directly:

```suggestion
    if (conversationId) {
      router.push('/conversations/{conversationId}', {
        path: {
          conversationId,
        },
        query: {},
      });
    } else {
      router.push('/conversations', { path: {}, query: {} });
    }
```

Btw we should make `path` and `query` optional when empty.

Reply:
It is not possible to navigate to another app without using this method.